VenafiPS/Public/Invoke-TppCertificateRenewal.ps1

```diff
@@ -8,17 +8,18 @@ immediate renewal. The certificate must not be in error, already being processed
 configured for Monitoring in order for it be renewable. You must have Write access
 to the certificate object being renewed.
 
-.PARAMETER InputObject
```

Comment:
InputObject was used for pipelining purposes of type TppObject.  I added ValueFromPipelineByPropertyName to the Path property so will have the same effect.  Just FYI, your comment was on the help and not the code so removing the line you mentioned wouldn't affect functionality per se.

Reply:
If I want to deprecate a parameter, I will typically put a write-warning statement letting folks know it will be in the future and support it for now.

Reply:
Good to know, thanks